frontend/package.json

```diff
@@ -35,5 +37,10 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:8080"
+  "proxy": "http://localhost:8080",
+  "devDependencies": {
+    "@material-ui/core": "^4.11.3",
+    "axios": "^0.21.1",
+    "react-router-dom": "^5.2.0"
```

Comment:
none of these should be dev dependencies as they are probably intended for use in your production code as well